src/components/play/PlayHints.tsx

```diff
@@ -0,0 +1,42 @@
+import { useTranslation } from 'react-i18next';
+
+import { Alert, AlertTitle } from '@mui/material';
+
+import { HINTS_PLAY_CY } from '../../config/selectors';
+import { QUIZ_TRANSLATIONS } from '../../langs/constants';
+
+type Props = {
+  hints: string | undefined;
+  maxAttempts: number;
+  currentAttempts: number;
+  isCorrect: boolean;
+  mt?: number;
```

Comment:
Mhmm not sure it makes sens to have the `marginTop` as a prop. Maybe allow to `sx` and spread it with your default style.